modules/express/src/errors.ts

```diff
@@ -40,10 +40,3 @@ export class IpcError extends Errors.BitGoJsError {
     Object.setPrototypeOf(this, IpcError.prototype);
   }
 }
-
-export class ExternalSignerConfigError extends Errors.BitGoJsError {
```

Comment:
I'm curious why we're getting rid of this error in favor of throwing the top level `Error` type instead?

Reply:
Ah okay I realize now that I misunderstood this original comment. https://github.com/BitGo/BitGoJS/pull/1944#discussion_r798135161 

I believe you meant that just `Object.setPrototypeOf(this, ExternalSignerConfigError.prototype);` was unnecessary. I'll edit this commit.